src/App.tsx

```diff
@@ -165,38 +201,120 @@ class Board extends React.Component<BoardProps, BoardState> {
         }
     }
 
+    public handleGlobalKeyUp = (e: KeyboardEvent) => {
+        if (e.keyCode === 91) {
+            e.preventDefault();
+            const newState = _.cloneDeep(this.state);
+            newState.continueHighlighting = false;
+            this.setState(newState);
+        }
+    }
+
+    public handleGlobalMouseDown = (e: MouseEvent) => {
+        if (!e.defaultPrevented && !this.state.continueHighlighting) {
```

Comment:
Seems like you can drop `defaultPrevented` in favor of `stopPropagation`

Reply:
I don't want to stop propagation due to the chance that later down the line I'll need to process these Mouse events in some other way. So it shouldn't make a difference using `defaultPrevented` / `preventDefault` vs `stopPropogation`

Reply:
right?